nitransforms/io/afni.py

```diff
@@ -108,17 +108,24 @@ def from_string(cls, string):
         sa["parameters"] = parameters
         return tf
 
-    def to_ras(self, moving=None, reference=None):
+    def to_ras(self, moving=None, reference=None, pre_rotation=None, post_rotation=None):
```

Comment:
I would just revert the changes to this function. You could unify it with what we added below, if you want to spend the effort.